### ISSUE TITLE
FindElements is not working for anything other than AccessibilityId.

### ISSUE THREAD

**Issue by Taldren76** [microsoft/WinAppDriver#1276]
The only 'FindElements' that seems to work is **FindElementsByAccessibilityId**. All other Find Elements fail to find elements that can be found by 'FindElement' using the exact same argument. See below for an example.

My code:

```
                WindowsElement test = DB.FindElementByXPath("//*[contains(@Name,'Owner')and @LocalizedControlType='link']") as WindowsElement;
                var tests = DB.FindElementsByXPath("//*[contains(@Name,'Owner')and @LocalizedControlType='link']"); 
```
WinAppDriver console:

> POST /session/F3E66263-02DD-40D3-BC45-D6B1FB171A2C/element/42.460294/element HTTP/1.1
> Accept: application/json, image/png
> Content-Length: 88
> Content-Type: application/json;charset=utf-8
> Host: 127.0.0.1:4723
> User-Agent: selenium/3.141.0 (.net windows)
> 
> {"using":"xpath","value":"//*[contains(@Name,'Owner')and @LocalizedControlType='link']"}
> HTTP/1.1 200 OK
> Content-Length: 103
> Content-Type: application/json
> 
> {"sessionId":"F3E66263-02DD-40D3-BC45-D6B1FB171A2C","status":0,"value":{"ELEMENT":"42.198240.4.-1959"}}
> 
> 
> ==========================================
> POST /session/F3E66263-02DD-40D3-BC45-D6B1FB171A2C/element/42.460294/elements HTTP/1.1
> Accept: application/json, image/png
> Content-Length: 88
> Content-Type: application/json;charset=utf-8
> Host: 127.0.0.1:4723
> User-Agent: selenium/3.141.0 (.net windows)
> 
> {"using":"xpath","value":"//*[contains(@Name,'Owner')and @LocalizedControlType='link']"}
> HTTP/1.1 200 OK
> Content-Length: 74
> Content-Type: application/json
> 
> {"sessionId":"F3E66263-02DD-40D3-BC45-D6B1FB171A2C","status":0,"value":[]}

**Comment by anunay1** [comment]
The response that you get is 200, that means it's, I have used find elements in two projects other than accessibility id, and it works perfectly what are you trying to achieve

**Comment by Taldren76** [comment]
> The response that you get is 200, that means it's, I have used find elements in two projects other than accessibility id, and it works perfectly what are you trying to achieve

The FindElements response...
>{"sessionId":"F3E66263-02DD-40D3-BC45-D6B1FB171A2C","status":0,"value":[]}
returns [] values, meaning nothing has been returned from the query. The value of tests is null.

Yet the FindElement reponse, using the same argument, returned...
>{"sessionId":"F3E66263-02DD-40D3-BC45-D6B1FB171A2C","status":0,"value":{"ELEMENT":"42.198240.4.-1959"}}
meaning that the FindElements query should have returned at least this one result as well.
